torchinductor/ir.py

```diff
@@ -3007,7 +3013,13 @@ def create(
                 x.get_dtype(),
             )
         else:
-            output_layout_str = "torch.contiguous_format"
+            # if x or weight have one channels_last format, it will call channels_last path,
+            # which align with aten.convolutuion path(only support 2d case).
+            if len(x.get_size()) == 4 and (x.get_layout().is_channels_last_stride_ordered() or \
```

Comment:
Please add TODO here as it does not support conv3d now.

Reply:
Yes, added.